common/static/common/js/common_libraries.js

```diff
@@ -1 +1,19 @@
-define(['domReady!', 'jquery', 'backbone', 'underscore', 'gettext']);
+define([
```

Comment:
Reviewers: it turns out that string_utils is not available in Studio, but @cahrens pointed out that this common_libraries file is where other global namespace variables are set up (primarily ``$`` and ``_``). The nice thing about this solution is that it isn't using a clone of the UI Toolkit code, but is just bringing it in directly as any code would.